environments/development/common/cloudfront.tf

```diff
@@ -282,7 +282,7 @@ module "backups_cdn" {
       target_origin_id       = "dumps"
       viewer_protocol_policy = "redirect-to-https"
 
-      cache_policy_id            = data.aws_cloudfront_cache_policy.caching_disabled.id 
+      cache_policy_id            = data.aws_cloudfront_cache_policy.caching_disabled.id
```

Comment:
Snap lol

Reply:
Looks like `terraform fmt` picked this one up

Reply:
Yeah, same as in my branch was the snap :)